tests/src/betree/aeneas-test-options

```diff
@@ -0,0 +1,4 @@
+charon-args=--polonius --opaque=betree_utils
+aeneas-args=-backward-no-state-update -test-trans-units -state -split-files
+[coq] aeneas-args=-use-fuel
+[fstar] aeneas-args=-decreases-clauses -template-clauses
```

Comment:
This file is needed because the betree is a crate. But actually we can turn it into a single file.

Reply:
It is a crate today because it has dependencies (notably serde). Do we want to give up on that?